src/plugins/intel_cpu/tests/unit/snippets_transformations/aarch64/lowered/gemm_blocking.cpp

```diff
@@ -0,0 +1,209 @@
+    }
+    {
+        auto data_a = linear_ir_ref->push_node<ov::opset10::Parameter>(precision, input_shape_a);
+        auto data_b = linear_ir_ref->push_node<ov::opset10::Parameter>(precision, input_shape_b);
+        auto gemm = linear_ir_ref->push_node<aarch64::GemmCPU>(data_a.second,
+                                                               data_b.second,
+                                                               PortDescriptor{},
+                                                               PortDescriptor{},
+                                                               PortDescriptor{0, 0},
+                                                               layout_a,
+                                                               layout_b,
+                                                               layout_c);
+        const auto& gemm_expr = *gemm.first;
+        init_expr_descriptors(gemm_expr, {{m_blk, full_dim}, {full_dim, n_blk}, {m_blk, n_blk}}, {layout_a, layout_b, layout_c});
+        create_gemm_loop_infos(linear_ir_ref, gemm_expr, 384, m_blk, 0, 0, 384, n_blk);
+        gemm_expr->set_loop_ids({1, 0});
+        auto result = linear_ir_ref->push_node<ov::opset10::Result>(gemm.second);
+    }
+}
+
+TEST_F(GemmCPUBlockingTest, Floating_LargeK) {
+    const ov::Dimension::value_type m = 384;
+    const ov::Dimension::value_type n = 384;
+    const ov::Dimension::value_type k = 2048;
+    const ov::PartialShape input_shape_a{1, 16, m, k};
+    const ov::PartialShape input_shape_b{1, 16, k, n};
+    const auto precision = ov::element::f32;
+    k_blk = full_dim;
+
+    {
+        auto data_a = linear_ir->push_node<ov::opset10::Parameter>(precision, input_shape_a);
+        auto data_b = linear_ir->push_node<ov::opset10::Parameter>(precision, input_shape_b);
+        auto gemm = linear_ir->push_node<aarch64::GemmCPU>(data_a.second, data_b.second,
+                                                           PortDescriptor{}, PortDescriptor{}, PortDescriptor{});
+        init_expr_descriptors(*gemm.first, {});
+        auto result = linear_ir->push_node<ov::opset10::Result>(gemm.second);
+    }
+    {
+        auto data_a = linear_ir_ref->push_node<ov::opset10::Parameter>(precision, input_shape_a);
+        auto data_b = linear_ir_ref->push_node<ov::opset10::Parameter>(precision, input_shape_b);
+        auto gemm = linear_ir_ref->push_node<aarch64::GemmCPU>(data_a.second, data_b.second,
+                                                               PortDescriptor{}, PortDescriptor{}, PortDescriptor{});
+        const auto& gemm_expr = *gemm.first;
+        init_expr_descriptors(gemm_expr, {{m_blk, full_dim}, {full_dim, n_blk}, {m_blk, n_blk}});
+        create_gemm_loop_infos(linear_ir_ref, gemm_expr, m, m_blk, 0, 0, n, n_blk);
+        gemm_expr->set_loop_ids({1, 0});
+        auto result = linear_ir_ref->push_node<ov::opset10::Result>(gemm.second);
+    }
+}
+
+TEST_F(GemmCPUBlockingTest, Float_FC) {
```

Comment:
FC scenario doesn't differ from matmul on ARM platform, so this test case is redundant

Reply:
Removed